### ISSUE TITLE
elpy-config resets current virtualenv on Windows

### ISSUE THREAD

**Issue by Holt59** [jorgenschaefer/elpy#1748]
# Summary

When using the `elpy-config` command on Windows, the currently activated virtualenv is deactivated.

_I also think that most of `elpy-` commands do not consider the currently activated virtual environment because some environment-specific package are not found (e.g., by `elpy-doc`)._

# Steps to reproduce

1. Open a python file in a buffer.
2. Activate a virtual environment using `pyvenv-workon`.
3. Run `elpy-config`
     - At this point, `elpy-config` is not in the virtual environment anymore.
4. Go back to the original buffer.
     - The current virtual environment is now the default one, not the one activated in step 2.

# My configuration
## OS

I am on Windows with a Python 3.8 installation (not Anaconda).

## Result of `(elpy-config)`
```
Elpy Configuration

Emacs.............: 26.3
Elpy..............: 1.32.0
Virtualenv........: Python (c:/Users/mikael.capelle/AppData/Local/Programs/Python)
Interactive Python: ipython (not found)
RPC virtualenv....: rpc-venv (c:/Users/mikael.capelle/.emacs.d/elpy/rpc-venv)
 Python...........: c:/Users/mikael.capelle/AppData/Local/Programs/Python/Launcher/py.exe 3.8.1 (c:/Users/mikael.capelle/AppData/Local/Programs/Python/Launcher/py.exe)
 Jedi.............: 0.15.2
 Rope.............: 0.16.0
 Autopep8.........: Not found (1.4.4 available)
 Yapf.............: Not found (0.29.0 available)
 Black............: 19.10b0
Syntax checker....: flake8.exe (c:/Softwares/Python/Python38/Scripts/flake8.exe)
```

## Elpy configuration in my init.el
```
(use-package elpy
  :defer t
  :bind
  (("M-/" . elpy-company-backend))
  :custom
  (flycheck-python-flake8-executable "python")
  (python-indent-offset 4)
  (python-shell-interpreter "ipython")
  (python-shell-interpreter-args "-i --simple-prompt")
  :config
  (setq elpy-rpc-backend "jedi")
  (setq elpy-modules (delq 'elpy-module-flymake elpy-modules))
  (add-hook 'elpy-mode-hook 'flycheck-mode)
  :init
  (advice-add 'python-mode :before 'elpy-enable))
```

**Comment by Holt59** [comment]
I did some investigation and apparently the culprit is `with-elpy-rpc-virtualenv-activated` which does not reactivate the current virtual environment.

**Comment by galaunay** [comment]
It looks like a typo indeed,
I fixed it in 46eb5a67429fbdaacad0bf6533289b05445b2108.

Thanks for investigating it.
